TheMovieDb/Network/MovieClient.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  MdbAPI.swift
+//  TheMovieDb
+//
+//  Created by Alex on 11/4/21.
+
+import Foundation
+import os.log
```

Comment:
I don't see where are we using the logger in this file, maybe we can omit this import.

Reply:
Got it.